### DISCUSSION TITLE
stimulus_confetti makes it easy to add confetti you your app

### DISCUSSION THREAD

**Discussion by adrianthedev** [therubyradar/submissions#99] [Submissions]
https://github.com/avo-hq/stimulus-confetti is the easiest way to add confetti to your StimulusJS app 🙌

Thank you!

~PS: We also launched https://github.com/avo-hq/class_variants a few weeks back which makes it simple to work with class-based components in ruby (I forgot if I submitted it in the past).~ it was published a while back. Thank you for that!

**Comment by andrewmcodes** [comment]
this is cool 😄

**Reply by adrianthedev** [reply]
Thanks!
